troop/forms.py

```diff
@@ -47,15 +41,54 @@ class Meta:
             "comment",
         ]
         widgets = {
-            "troop": HiddenInput,
-            "gender": RadioSelect,
-            "birthday": DateInput(format="%Y-%m-%d", attrs={"type": "date"}),
-            "age_section": RadioSelect(attrs={"required": True}),
-            "attendance": CheckboxSelectMultiple,
-            "diet": CheckboxSelectMultiple,
-            "comment": Textarea(attrs={"cols": 32, "rows": 4}),
+            "troop": forms.HiddenInput,
+            "gender": forms.RadioSelect,
+            "birthday": forms.DateInput(format="%Y-%m-%d", attrs={"type": "date"}),
+            "age_section": forms.RadioSelect(attrs={"required": True}),
+            "attendance": forms.CheckboxSelectMultiple,
+            "diet": forms.CheckboxSelectMultiple,
+            "comment": forms.Textarea(attrs={"cols": 32, "rows": 4}),
         }
 
 
-class NamiSearchForm(Form):
+class NamiSearchForm(forms.Form):
     nami = Participant._meta.get_field("nami").formfield()
+
+
+class SendEmailForm(forms.Form):
+    SECTION_CHOICES = (
+        ("all", _("All sections")),
+        (Participant.SECTION_BEAVER, _("Beavers")),
+        (Participant.SECTION_CUB, _("Cubs")),
+        (Participant.SECTION_SCOUT, _("Scouts")),
+        (Participant.SECTION_VENTURER, _("Venturers")),
+        (Participant.SECTION_ROVER, _("Rovers")),
+        ("", _("No section")),
+    )
+
+    STATUS_CHOICES = (
+        ("all", _("All")),
+        ("leaders", _("Only leaders")),
+        ("children", _("Only children")),
+    )
+
+    sender = forms.CharField(label=_("Sender"), disabled=True)
+    reply_to = forms.CharField(label=_("Reply to"))
```

Comment:
We should have some e-mail validation there
```suggestion
    reply_to = forms.EmailField(label=_("Reply to"))
```